Sources/SwiftGen/main.swift

```diff
@@ -47,6 +47,10 @@ let main = Group {
     }
   }
 
+  $0.group("xcfilelist", "manage xcfilelists") {
```

Comment:
Did a quick check of the current list of groups/commands (should have done this before 🤦 ), maybe it'd be better under the `config` group? Like `swiftgen config generate-xcfilelists`?